sweet.py

```diff
@@ -0,0 +1 @@
+# stuff
```

Comment:
What is that supposed to do???

Reply:
nothin really, I just thought it was pretty :D

Reply:
but it looks good homie